### PR TITLE
Kubernetes like issue for this repository

### PR DESCRIPTION
**Pull request by akanex115**
# Background
https://github.com/ajktown/api/issues/74

## TODOs
- [x] Apply the similar templates for this repository.

## Checklist (Developers)
- [x] `Draft` is set for this PR
- [x] `Title` is checked
- [x] `Background` is filled
- [x] `TODOs` are filled
- [x] `Assignee` is set
- [x] `Labels` are set
- [x] `Project` is created & linked, if multiple PRs are associated to this PR
- [x] `development` is linked if related issue exists
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
- [x] Check if there are any other missing TODOs that are not yet listed
- [x] Review Code
- [x] Every item on the checklist has been addressed accordingly
- [x] If `development` is associated to this PR, you must check if every TODOs are handled

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/dependency.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
+
```
Comment (mlajkim) [inline .github/ISSUE_TEMPLATE/dependency.yaml:24]: image
?

File: .github/ISSUE_TEMPLATE/documentation.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
+
```
Comment (mlajkim) [inline .github/ISSUE_TEMPLATE/documentation.yaml:24]: image
?

File: .github/ISSUE_TEMPLATE/infrastructure.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
+
```
Comment (mlajkim) [inline .github/ISSUE_TEMPLATE/infrastructure.yaml:24]: image
?

File: .github/ISSUE_TEMPLATE/performance.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
+
```
Comment (mlajkim) [inline .github/ISSUE_TEMPLATE/performance.yaml:24]: image
?

File: .github/ISSUE_TEMPLATE/refactor.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
+
```
Comment (mlajkim) [inline .github/ISSUE_TEMPLATE/refactor.yaml:24]: image
?

**Review by mlajkim — COMMENTED** [review]
review

File: .github/ISSUE_TEMPLATE/dependency.yaml (hunk 1 above)
Comment (akanex115) [reply .github/ISSUE_TEMPLATE/dependency.yaml]: Handled in [28c1773](https://github.com/ajktown/api/pull/83/commits/28c1773a512c5850d6cba3c712644a30cf5b59b7)

File: .github/ISSUE_TEMPLATE/documentation.yaml (hunk 2 above)
Comment (akanex115) [reply .github/ISSUE_TEMPLATE/documentation.yaml]: Handled in [28c1773](https://github.com/ajktown/api/pull/83/commits/28c1773a512c5850d6cba3c712644a30cf5b59b7)

File: .github/ISSUE_TEMPLATE/infrastructure.yaml (hunk 3 above)
Comment (akanex115) [reply .github/ISSUE_TEMPLATE/infrastructure.yaml]: Handled in [28c1773](https://github.com/ajktown/api/pull/83/commits/28c1773a512c5850d6cba3c712644a30cf5b59b7)

File: .github/ISSUE_TEMPLATE/performance.yaml (hunk 4 above)
Comment (akanex115) [reply .github/ISSUE_TEMPLATE/performance.yaml]: Handled in [28c1773](https://github.com/ajktown/api/pull/83/commits/28c1773a512c5850d6cba3c712644a30cf5b59b7)

File: .github/ISSUE_TEMPLATE/refactor.yaml (hunk 5 above)
Comment (akanex115) [reply .github/ISSUE_TEMPLATE/refactor.yaml]: Handled in [28c1773](https://github.com/ajktown/api/pull/83/commits/28c1773a512c5850d6cba3c712644a30cf5b59b7)

**Review by mlajkim — APPROVED** [review]
Good work

**Pull request merged**